scaleway/resource_function_cron.go

```diff
@@ -138,6 +144,7 @@ func resourceScalewayFunctionCronUpdate(ctx context.Context, d *schema.ResourceD
 	req := &function.UpdateCronRequest{
 		Region: region,
 		CronID: cron.ID,
+		Name:   expandStringPtr(d.Get("name")),
```

Comment:
Should be done with the same format as below to avoid sending the name in the update request when there is no changes
```
if d.HasChange("name") ...
```